registry.k8s.io/README.md

```diff
@@ -124,3 +124,23 @@ requests, which is described in detail
 [post-promo-job]: https://testgrid.k8s.io/sig-release-releng-blocking#post-k8sio-image-promo
 [ci-promo-job]: https://testgrid.k8s.io/sig-release-releng-blocking#ci-k8sio-image-promo
 [project-github]: https://git.k8s.io/community/github-management#project-owned-organizations
+
+### Publishing helm charts on registry.k8s.io
+
+Helm charts are a popular way to install Kubernetes applications.
```

Comment:
We don't need this line. (I'm not disagreeing, but it doesn't add detail and this doc is already long)